### ISSUE TITLE
"ignoring emit path because multiple .o files were produced" false positive

### ISSUE THREAD

**Issue by ishitatsuyuki** [rust-lang/rust#49801]
Minimal reproduction invocation:

```
rustc src/lib.rs --crate-type lib --emit=obj -C incremental=target/debug/incremental
```

```
warning: ignoring emit path because multiple .o files were produced
```

It seems that it assumes that the path is set somewhere although we don't. The corresponding code is here:

https://github.com/rust-lang/rust/blob/8ae79efce3e43eadecd032cf38b2372b4cba7b62/src/librustc_trans/back/write.rs#L1107

**Comment by Enselic** [comment]
The reproducer does not work any longer for me. Can you still reproduce this?

**Comment by Enselic** [comment]
Triage: Closing for now. Feel free to reopen with updated instructions on how to reproduce.
